Lake/DSL/Script.lean

```diff
@@ -11,17 +11,17 @@ namespace Lake.DSL
 open Lean Parser Command
 
 syntax scriptDeclSpec :=
-  ident (ppSpace "(" Term.simpleBinder ")")? (declValSimple <|> declValDo)
+  ident (ppSpace "(" ident+ ")")? (declValSimple <|> declValDo)
```

Comment:
I'm assuming it was not really intended to accept either holes or a type ascription here

Reply:
Actually, it was. 😅 The reason why I used a binder rather than just an ident was explicitly allow users to add a type ascription for readable reasons (i.e., so they could `(args : List String)` if they desired to make it clear to readers of the code what type `args` is).

Reply:
I see. But the repetition at least seems to be unnecessary then.